django/db/migrations/operations/special.py

```diff
@@ -45,13 +45,16 @@ def database_forwards(self, app_label, schema_editor, from_state, to_state):
 
     def database_backwards(self, app_label, schema_editor, from_state, to_state):
         # We calculate state separately in here since our state functions aren't useful
-        base_state = to_state
-        for pos, database_operation in enumerate(reversed(self.database_operations)):
-            to_state = base_state.clone()
-            for dbop in self.database_operations[:-(pos + 1)]:
-                dbop.state_forwards(app_label, to_state)
-            from_state = to_state.clone()
-            database_operation.state_forwards(app_label, from_state)
+        to_states = {}
+        for dbop in self.database_operations:
+            to_states[dbop] = to_state
+            to_state = to_state.clone()
+            dbop.state_forwards(app_label, to_state)
+        # to_state now has the final db state of the forwards migration, hence
```

Comment:
No, the original comment is correct. This is the state after applying the states of all the database_operations.
Perhaps
```
# to_state now has the states of all the database_operations applied, ...
```